pkg/platform/baremetal/baremetal.go

```diff
@@ -60,6 +60,11 @@ type config struct {
 	KubeAPIServerExtraFlags  []string
 }
 
+const (
+	// Name represents bare-metal platform name as it should be referenced in function calls and configuration.
```

Comment:
We should be consistent like other platforms.

```suggestion
      // Name represents Bare-metal platform name as it should be referenced in function calls and configuration.
```

Reply:
True, but it should be `Bare Metal`, as in quickstart guide.